modules/iswa/rendering/iswakameleongroup.cpp

```diff
@@ -180,11 +179,10 @@ void IswaKameleonGroup::clearFieldlines() {
         if (std::get<2>(seedPath.second)) {
             LDEBUG("Removed fieldlines: " + std::get<0>(seedPath.second));
 
-            global::scriptEngine->queueScript(
-                "openspace.removeSceneGraphNode('" + std::get<0>(seedPath.second) + "')",
-                scripting::ScriptEngine::ShouldBeSynchronized::Yes,
-                scripting::ScriptEngine::ShouldSendToRemote::Yes
+            const std::string script = std::format(
+                "openspace.removeSceneGraphNode('{}')", std::get<0>(seedPath.second)
             );
+            global::scriptEngine->queueScript({ script });
```

Comment:
It's a bit annoying that we have to add these brackets for every `queueScript` now... How about making a version of the `queueScript` that takes just a string and does this for us ?

It is the most common use case, after all.

Reply:
Very good point.  Changed that